share/models/ingest.py

```diff
@@ -73,6 +73,14 @@ class Source(models.Model):
     icon = models.ImageField(upload_to=icon_name, storage=SourceIconStorage(), blank=True)
     is_deleted = models.BooleanField(default=False)
 
+    # Whether or not this SourceConfig collects original content
+    # If True changes made by this source cannot be overwritten
```

Comment:
I suppose `cannot be overwritten` should now say `cannot be overwritten by a non-canonical source`